core/src/storage/device.py

```
@@ -668,13 +673,32 @@ def get_wp_cnts() -> int:
+        _FIDO2_COUNTER_VALUE = (
+            int.from_bytes(counter, "big") if counter is not None else 0
+        )
+
+    return _FIDO2_COUNTER_VALUE
+
+
+def set_fido2_counter(value: int) -> None:
+    global _FIDO2_COUNTER_VALUE
+    from .resident_credentials import MAX_RESIDENT_CREDENTIALS
+
+    assert (
```

Ruff (0.8.2)

Use of `assert` detected

(S101)